src/sql/workbench/api/node/mainThreadConnectionManagement.ts

```diff
@@ -60,8 +60,12 @@ export class MainThreadConnectionManagement implements MainThreadConnectionManag
 	}
 
 	public async $openConnectionDialog(providers: string[], initialConnectionProfile?: IConnectionProfile, connectionCompletionOptions?: azdata.IConnectionCompletionOptions): Promise<azdata.connection.Connection> {
+		let connectionType: number = 1;
```

Comment:
can we not use ConnectionType.editor

Reply:
No, because if it's an editor connection, it expects an `input` from the params as well

Reply:
I think she's asking - and I agree with - is why aren't you using the enum you made in connectionManagement.ts? Hardcoded numbers like this are bleh. 

---
In reply to: [285718562](https://github.com/microsoft/azuredatastudio/pull/5542#discussion_r285718562) [](ancestors = 285718562)